### ISSUE TITLE
What is the mobilenet.caffemodel  deploy.prototxt output ? There no output?

### ISSUE THREAD

**Issue by siminghua** [shicai/MobileNet-Caffe#22]
I have run the mobilenet.caffemodel deploy.prototxt with SSD caffe to detect image but no output .

**Comment by siminghua** [comment]
How to use the mobilenet.caffemodel to detect the image?
I0720 16:15:52.949961  5313 net.cpp:270] This network produces output prob
I0720 16:15:52.950126  5313 net.cpp:283] Network initialization done.
I0720 16:15:53.019024  5313 upgrade_proto.cpp:77] Attempting to upgrade batch norm layers using deprecated params: models/VGGNet/VOC0712/SSD_300x300/mobilenet.caffemodel
I0720 16:15:53.019157  5313 upgrade_proto.cpp:80] Successfully upgraded batch norm layers using deprecated params.
I0720 16:15:53.019214  5313 net.cpp:761] Ignoring source layer data
I0720 16:15:53.019255  5313 net.cpp:761] Ignoring source layer label_data_1_split
I0720 16:15:53.026746  5313 net.cpp:761] Ignoring source layer fc7_fc7_0_split
I0720 16:15:53.026836  5313 net.cpp:761] Ignoring source layer loss
I0720 16:15:53.026871  5313 net.cpp:761] Ignoring source layer top1/acc
I0720 16:15:53.026991  5313 net.cpp:761] Ignoring source layer top5/acc

**Comment by shicai** [comment]
This pretrained model is used for classification, and cannot directly be used for detection.
And also, please make sure that you are familiar with caffe.

**Comment by siminghua** [comment]
Thanks
